src/components/atoms/definition-list/definition-list.scss

```diff
@@ -1,41 +1,52 @@
 @import "~/styles/000-abstracts/__abstracts-dir";
+
 .definition-list {
-  display: flex;
+  //display: flex;
```

Comment:
Sollen wir bei CodeReviews darauf achten, dass keine auskommentierten Codezeilen mit-gemerged werden?

Es gibt leider ein paar Stellen, wo ein paar Codezeilen auskommentiert in Commits geführt werden.

@cnoss Wie siehst du das?

Reply:
hin und her gerissen … gerade bei größeren Umbauarbeiten ist es manchmal hilfreich/ praktisch den alten code zu erhalten, bis der Umbau fertig ist … kannst Du entscheiden

Reply:
Befürchte nur, dass nicht immer ganz klar ist, welche auskommentierten Codezeilen noch aktuell sind (Gedankenstütze für einen anderen Entwickler darstellen) und welche entfernt werden können.
Können wir aber erstmal drin belassen.

Sollten aber im Hinterkopf behalten, bei Codezeilen, die man wirklich nicht mehr brauchen wird, auch zu entfernen und nicht nur auszukommentieren.

Müssen dann ab einem bestimmten Punkt nochmal durch den Code schauen, und solche Altlasten wirklich entfernen, sofern sie nicht mehr benötigt werden. Oder direktes Cleanup, wenn man das Stück Code nochmal anfasst oder dort vorbeikommt.